set.mm

```diff
@@ -62489,20 +62489,45 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
 
   ${
     $d x A $.  $d x F $.
-    $( A function's value at a proper class is the empty set.  (Contributed by
-       NM, 20-May-1998.) $)
+    $( A function's value at a proper class is the empty set.  See ~ fvprcALT
+       for a proof that uses ~ ax-sep and ~ ax-pr instead of ~ ax-pow .
+       (Contributed by NM, 20-May-1998.) $)
     fvprc $p |- ( -. A e. _V -> ( F ` A ) = (/) ) $=
       ( vx cvv wcel wn cv wbr weu cfv c0 wceq brprcneu tz6.12-2 syl ) ADEFACGBH
       CIFABJKLCABMCABNO $.
   $}
 
+  ${
+    $d A x y $.  $d F x y $.
+    $( Alternate proof of ~ fvprc using ~ ax-sep and ~ ax-pr instead of
+       ~ ax-pow .  While this proof requires more axioms, it can still be
+       useful for eliminating dependencies on the Axiom of Power Sets in proofs
+       that already depend on Separation and Pairing.  (Contributed by
+       BTernaryTau, 3-Aug-2024.)  The "discouraged" tags are here only because
+       this is an *ALT result.  (Proof modification is discouraged.)
+       (New usage is discouraged.) $)
+    fvprcALT $p |- ( -. A e. _V -> ( F ` A ) = (/) ) $=
```

Comment:
Maybe we can call this theorem ~fvprc2 to avoid the suffix ALT and the "discouraged" tags.

Reply:
That would definitely make things a lot simpler. We could also create a different suffix to differentiate between identical theorems with different proofs/axioms and merely similar theorems. (If the existing suffix was something like DUP instead of ALT, I would suggest ALT as the new suffix.) Maybe something like fvprcAX2, or fvprcV2?

Reply:
I made a similar suggestion some years ago (I proposed suffix ALTV), but this was not accepted in general, but only for mathboxes (see "Conventions").